crates/pipeline-manager/src/api/pipeline.rs

```diff
@@ -218,7 +218,11 @@ pub(crate) async fn get_pipeline(
         (status = CONFLICT
             , description = "Cannot create pipeline as the name already exists"
             , body = ErrorResponse
-            , example = json!(examples::error_duplicate_name()))
+            , example = json!(examples::error_duplicate_name())),
```

Comment:
So errors that are not captured by the schema are not reported in a nice way? Is it possible to have some kind of a catch-all clause to print HTTP status + the body of the response?

Reply:
the reason I can't print the text of the response in this case is kind of brain dead.. the error handler closure would need to be async since response.text() from reqwest is an async function... and async closures are not stable are not stable.. I can try to refactor this code so maybe I don't need to make a closure there

Reply:
I see. This is a pretty big pain point for me when using fda: a lot of the time I've no idea what went wrong, especially when it is called from a script in CI. So I'd say it's only a matter of priority when to fix this. Your call.